lib/installV1.js

```diff
@@ -582,7 +592,7 @@ function questions(done) {
             message: "Do you want to expose Arango DB:",
             default: false,
             when: (values) => {
-               return !values.arangoExpose && !Options.arangoExpose;
+               return !values.arangoExpose && typeof(Options.arangoExpose) == "undefined"
```

Comment:
technically speaking: ``` values.arangoExpose ``` should be undefined at this point.  if however, there happens to be a ```false``` value, shouldn't we change this to also be:
```
return typeof(values.arangoExpose) == "undefined" && typos(....)
```

Reply:
hmm that's true... i'll change as recommended.  or should we just take out that check altogether?  i can't imagine how the code flow would get to a state where tie value is set at this point?

Reply:
let's leave in the check for now.

Reply:
ok, it's back in as of the latest commit, using `typeof == "undefined"`